packages/smooth_app/lib/cards/product_cards/smooth_product_card_found.dart

```diff
@@ -205,6 +204,8 @@ class SmoothProductCardFound extends StatelessWidget {
                     children: <Widget>[
                       SmoothProductImage(
                         product: product,
+                        width: MediaQuery.of(context).size.width * 0.25,
+                        height: 140.0,
```

Comment:
Not really responsive, this `140.0`...

Reply:
Thanks @monsieurtanuki. I replaced it to be now a square  again